docs/oid-info.md

```diff
@@ -58,11 +95,93 @@ the git ref that the workflow run was based upon.
 This specifies the username identity in the OtherName Subject Alternative Name, as
 defined by [RFC5280 4.2.1.6](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.6).
 
+
+### 1.3.6.1.4.1.57264.1.8 | Source Repository URI
+
+Should include a fully qualified source repository URL that the build was based
+on. For example: `https://github.com/sigstore/fulcio`.
+
+### 1.3.6.1.4.1.57264.1.9 | Source Repository Digest
+
+An immutable reference to a specific version of the source code that the build
+was based upon. Should include the digest type followed by the digest. For example:
+`sha1:abc123` for a git commit.
```

Comment:
Not to make things more difficult here (sorry!), but is there any possibility we could make this more normative?

For example, we could enforce the following:

1. Only certain digest types are valid (e.g. `sha1`, `sha256`)
2. The digest itself must be a hexadecimal string, not another representation (like base64)
3. The digest itself must be a full digest, not a truncated one

(This is a good argument IMO for using ASN.1 structures, since we'd get a lot of these restrictions for "free" with ASN.1 definitions. But that's further out, so just listing these as plain language constraints would be good IMO.)

Reply:
Our thought was that Fulcio should not be opinionated on claim values and it's up to the verifier to understand what claims will look like (i.e. they must consult the documentation of the provider, not Fulcio).

Reply:
That makes sense! I'm a big fan of ratcheting down formats/flexibility wherever it can be ratcheted, but I can understand if this is a place where provider variability/diversity would make doing so too complicated or onerous.

Given that, I think many of these claims would benefit from `MAY` language that emphasize that providers may choose to fill these values as they please, e.g.:

```
Providers MAY choose to emit this extension's value in other formats, and consumers MUST NOT assume the format above. 
```

Reply:
+1, I think we should be more clear in describing examples as optional suggestions.

We wanted to make it as easy as possible to onboard new providers, which was another justification for fulcio just forwarding provided claims. Otherwise we're requiring CI platforms to conform to fulcio's requirements, which would likely not happen, or fulcio would mutate the claim before certifying it, which increases trust in fulcio.

Reply:
> Otherwise we're requiring CI platforms to conform to fulcio's requirements, which would likely not happen, or fulcio would mutate the claim before certifying it, which increases trust in fulcio.

Oh, this is a great point -- I hadn't considered how this places additional trust in Fulcio. Thanks!

Reply:
> Given that, I think many of these claims would benefit from `MAY` language that emphasize that providers may choose to fill these values as they please, e.g.:

@woodruffw good idea! I added this as top-level call out to cover all the values.